pkg/ansible/proxy/proxy.go

```diff
@@ -234,14 +243,15 @@ type HandlerChain func(http.Handler) http.Handler
 // Options will be used by the user to specify the desired details
 // for the proxy.
 type Options struct {
-	Address          string
-	Port             int
-	Handler          HandlerChain
-	NoOwnerInjection bool
-	KubeConfig       *rest.Config
-	Cache            cache.Cache
-	RESTMapper       meta.RESTMapper
-	ControllerMap    *ControllerMap
+	Address           string
+	Port              int
+	Handler           HandlerChain
+	NoOwnerInjection  bool
+	KubeConfig        *rest.Config
+	Cache             cache.Cache
+	RESTMapper        meta.RESTMapper
+	ControllerMap     *ControllerMap
+	WatchedNamespaces map[string]interface{}
```

Comment:
I think the option should take a list of namespace, and the `Run` function should take this list and make the map. 

This will make everything easier for users calling the `Run` method as everything else (way olm passes the namespace, multi-namespace cache, this) will then take a list instead of this being an outlier.

Reply:
Thanks, should be done in the recent commit.